docs/Stream-API.md

```diff
@@ -0,0 +1,473 @@
+---
+title: Stream API Overview
+layout: documentation
+documentation: true
+---
+
+* [Concepts](#concepts)
+    * [Stream Builder] (#streambuilder)
+    * [Value mapper] (#valuemapper)
+* [Stream APIs](#streamapis)
+    * [Basic transformations] (#basictransformations)
```

Comment:
It seems the space does not matter. I am able to click on the links here - https://github.com/arunmahadevan/storm/blob/STORM-2367/docs/Stream-API.md